CONTRIBUTING.md

```diff
@@ -15,6 +15,12 @@ python3 -m venv venv
 ./venv/bin/pip install -e '.[dev]'
 ```
 
+*Note:*
+Sygnal depends on the `psycopg2` database adapter for PostgreSQL. It is recommended that you install `libq-dev python3-dev build-essential` as root, i.e
+
+```sudo apt install libpq-dev python3-dev build-essential```
+
+
```

Comment:
A few requests:

* could you wrap the text at about 80 characters, in the same way as the rest of the document? Doing so makes it easier to read the raw markdown, and makes it easier to make changes in future.
* For the ``` styling to work properly, you need to use separate lines - see the other examples.
* You need to do this *before* running `pip install`, so I think it would make sense to put it further up in the text.
* This command only works on Debian/Ubuntu, so you might say something like "... which requires the libpq headers and libraries. For example, on Debian/Ubuntu, these can be installed with ..."

Reply:
Hi, thanks for the suggestions. On it